backend/lib/endpoints/schema/posts.js

```diff
@@ -1,48 +1,84 @@
 const S = require("fluent-schema");
 
+const { join } = require("path");
+const PATH_TO_SCHEMAS = join(__dirname, "../../models/schemas/v2");
+const { schema: postSchema } = require(join(PATH_TO_SCHEMAS, "post"));
+const { schema: orgSchema } = require(join(PATH_TO_SCHEMAS, "individualUser"));
+const { schema: userSchema } = require(join(
+  PATH_TO_SCHEMAS,
+  "organizationUser",
+));
+
+const EXPIRATION_OPTIONS = ["day", "week", "month", "forever"];
+const POST_OBJECTIVES = postSchema.tree.objective.enum;
+const POST_TYPES = postSchema.tree.types.enum;
+const USER_TYPES = [].concat(
+  userSchema.tree.type.enum,
+  orgSchema.tree.type.enum,
+);
```

Comment:
Is there any need to use `.concat` here? Maybe change it to use the array literal
```js
const USER_TYPES = [userSchema.tree.type.enum,  orgSchema.tree.type.enum];
```